### PR TITLE
feature/ORCA-171

### PR DESCRIPTION
**Pull request by rizbi-hassan**
## Summary of Changes

Addresses [ORCA-171: Use new shared_recovery shared library in copy_files_to_archive lambda function](https://bugs.earthdata.nasa.gov/browse/ORCA-171)

## Changes

* added build.sh and run_tests.sh under tasks/copy_files_to_archive/bin to run the unit tests
* modified  tasks/copy_files_to_archive/copy_files_to_archive.py to use the new shared library. 
* modified  tasks/copy_files_to_archive/test/unit_tests/test_copy_files_to_archive.py to use the new shared library. Unit test passed 96%. Make sure to change your shared_recovery.py to use the new one since it is not yet merged into develop branch, orelse the test will fail.

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Under tasks/copy_files_to_archive, ran bin/run_tests.sh and the test passed with 96% coverage.
Under tasks/copy_files_to_archive, ran bin/build.sh and the test passed and created the zip file.

### CODE REVIEW

File: tasks/copy_files_to_archive/bin/build.sh [hunk 1]
```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```
Comment (AndrewDorn) [inline tasks/copy_files_to_archive/bin/build.sh:1]: Remember to modify the build/test scripts in root/bin to point to new scripts.

File: tasks/copy_files_to_archive/bin/run_tests.sh [hunk 2]
```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```
Comment (AndrewDorn) [inline tasks/copy_files_to_archive/bin/run_tests.sh:1]: Remember to modify the build/test scripts in root/bin to point to new scripts.

File: tasks/copy_files_to_archive/copy_files_to_archive.py [hunk 3]
```diff
@@ -105,92 +86,19 @@ def task(records: List[Dict[str, Any]], max_retries: int, retry_sleep_secs: floa
...
-            post_status_for_file_to_queue(
-                a_file[INPUT_JOB_ID_KEY], a_file[INPUT_GRANULE_ID_KEY], a_file[INPUT_FILENAME_KEY], None,
-                None,
-                ORCA_STATUS_FAILED, a_file.get(FILE_ERROR_MESSAGE_KEY, None),
-                None, now, now, RequestMethod.PUT, db_queue_url,
-                max_retries, retry_sleep_secs)
-
+            shared_recovery.update_status_for_file(
+                a_file[INPUT_JOB_ID_KEY], a_file[INPUT_GRANULE_ID_KEY], a_file[INPUT_FILENAME_KEY], 
+                shared_recovery.OrcaStatus.FAILED.value, a_file.get(FILE_ERROR_MESSAGE_KEY, None),db_queue_url)
```
Comment (AndrewDorn) [inline tasks/copy_files_to_archive/copy_files_to_archive.py:91]: In my nitpicky vein, there's a space missing in these params before db_queue_url.

**Review by AndrewDorn — CHANGES_REQUESTED** [review]
New scripts need to be added to higher scripts, but looks good in general.

**Review by bhazuka — CHANGES_REQUESTED** [review]
Per @AndrewDorn comments please update bin/build_tasks.sh and bin/run_tests.sh scripts in the repo base directory.

File: tasks/copy_files_to_archive/copy_files_to_archive.py (hunk 3 above)
Comment (rizbi-hassan) [reply tasks/copy_files_to_archive/copy_files_to_archive.py:91]: good catch :)

**Review by AndrewDorn — APPROVED** [review]

**Review by bhazuka — APPROVED** [review]

**Pull request merged**